bids-validator/bids_validator/rules/session_level_rules.json

```
@@ -156,7 +160,9 @@
         "_electrodes.json",
         "_ieeg.json",
         "_coordsystem.json",
-        "_photo.jpg"
+        "_photo.jpg",
+        "_photo.png",
+        "_photo.tif"
```

```suggestion
        "_photo.jpg"
```
Doesn't apply to "ieeg".

---

Done